### ISSUE TITLE
Gnomish Battle Chicken doesnt apply buff

### ISSUE THREAD

**Issue by jpaardekooper** [gamefreedomgit/Maelstrom#2744]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)
https://www.wowhead.com/item=10725/gnomish-battle-chicken

**Description:**
gnomish-battle-chicken spawns a chicken that attacks the enemy and has a chance to give you a buff that increases melee haste 

**How to reproduce:**
Right click on the trinket and watch your chicken fight an enemy and pray for him to get the Battle Squawk buff. But this never happens 

**How it should work:**
I haven't paid attention to individual attacks, but recount after a fight will show it having done ~2k DPS or so. Looking at the most recent WWS, individual attacks are only single digits, until the chicken gains [Chicken Fury](https://www.wowhead.com/?spell=13168). After it gained that buff, it started hitting for 75/159 crit. It also does still give [Battle Squawk](https://www.wowhead.com/?spell=23060), but only to party members, never raid members.

**Database links:**

**Comment by Vezax69** [comment]
There used to be a bug report that should've been already dismissed. It was hotfixed by the end of wrath and battle squawk it's not supossed to stack with any other melee haste effect. 

https://web.archive.org/web/20110509160316/http://elitistjerks.com/f76/t68951-retribution_updated_3_3_a/p91/#post1594392
Feel free to check the many comments that there are around like this:
''I picked up engineering and it is a noticable dps increase over BS, but the perk of Battle Chicken Squawk doesn't seem to work.
I tried it a few times, and got 15 Chicken Fury and 0 Squawks, but the Squawk buff did show up in WoL, so that implies to doesn't stack with WF.''
